src/Roslyn.Diagnostics.Analyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -2,6 +2,9 @@
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
 RS0040 | RoslynDiagnosticsReliability | Warning | DefaultableTypeShouldHaveDefaultableFieldsAnalyzer
+RS0043 | RoslynDiagnosticsMaintainability | Warning | DoNotCallGetTestAccessor
```

Comment:
Why not use RS0041 and RS0042?

Reply:
They are already taken

Reply:
Where? If they are taken, this file should have them listed. Is this in another open PR?

Reply:
RS0041 is part of Public API analyzer. RS0042 is Do Not Copy Value.